app/dashboard/management/commands/activity_report.py

```diff
@@ -212,13 +239,25 @@ def handle(self, *args, **options):
             created_on__lte=options['end_date']
         ).order_by('created_on', 'id')
         formted_enssubreg = imap(self.format_ens_reg, enssubregistrations)
-
+        
+        #grants contribution objcets
+        grantcontobjs = Contribution.objects.filter(
+            created_on__gte=options['start_date'],
+            created_on__lte=options['end_date']
```

Comment:
maybe filter for contributions on the mainnet ?

Reply:
If the suggestion works for you for now I'll clean it up and get it rolling. 🚢

Reply:
yes pls